aviary/interface/methods_for_level2.py

```diff
@@ -1604,8 +1604,10 @@ def add_driver(self, optimizer=None, use_coloring=None, max_iter=50, verbosity=V
         if verbosity != Verbosity.QUIET:
             if isinstance(verbosity, list):
                 driver.options['debug_print'] = verbosity
-            elif verbosity.value >= 2:
+            elif verbosity.value >= 3:
```

Comment:
For this one, "elif verbosity.value >= Verbosity.VERBOSE"
then turn on the driver debug.
Note, there are currently no options above VERBOSE, so you could just as well make it "== Verbosity.VERBOSE"

Reply:
we have 4 options quiet (0), brief (1), verbose (2), and debug (3)

Reply:
Sorry, I meant Verbosity.DEBUG everywhere I wrote VERBOSE above.